build.gradle

```diff
@@ -1418,7 +1418,7 @@ project(':connect:api') {
 
   javadoc {
     include "**/org/apache/kafka/connect/**" // needed for the `javadocAll` task
-    options.links "http://docs.oracle.com/javase/7/docs/api/"
+    options.links "https://docs.oracle.com/javase/7/docs/api/"
```

Comment:
@ijuma Is this correct?

Reply:
We updated this in other PRs. This needs to be rebased.

Reply:
@ijuma I did the rebase and tried to push after github didn't show any change, but it seems this has now included all commits here, I don't know why.